extensions/integration-tests/src/notebook.test.ts

```diff
@@ -14,136 +14,184 @@ import { sqlNotebookContent, writeNotebookToFile, sqlKernelMetadata, getFileName
 import { getBdcServer, getConfigValue, EnvironmentVariable_PYTHON_PATH } from './testConfig';
 import { connectToServer } from './utils';
 import * as fs from 'fs';
+import { stressify, sleep } from '../testfmks/src/stress';
 
 if (context.RunTest) {
 	suite('Notebook integration test suite', function () {
-		setup(function () {
+		setup(async function () {
+			sleep(20000);
+			console.log(`environment variable SuiteType is set to ${process.env.SuiteType}`);
```

Comment:
Why are you adding this?

Reply:
Actually I did not add this, Chris did when he was working in my branch probably to help him debug. So I will remove the sleep.